server/plugin/command.go

```diff
@@ -367,9 +412,44 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.BotUserID,
+		Message:   msg,
+	}
+
+	if _, appErr := p.API.CreatePost(post); appErr != nil {
+		p.API.LogWarn("error while creating post", "post", post, "error", appErr.Error())
+		return fmt.Sprintf("%s Though there was an error creating the public post: %s", msg, appErr.Error())
+	}
+
+	message := fmt.Sprintf("Successfully subscribed to [%s](%s) with events: %s.", repo, repoLink, subscribeEvents.String())
+	if previouslySubscribed {
+		message += previouslySubscribedEventMessage
+	}
+
+	return message
 }
+func (p *Plugin) getSubscribedEvent(channelID string, owner, repo string) (bool, Features, error) {
```

Comment:
Why is this methods called `getSubscribedEvent` if it returns a `Features`?

Reply:
That value of the `bool` return value?

Reply:
removed it, seems not necessary